x-pack/plugins/cloud_security_posture/public/components/compliance_score_bar.tsx

```diff
@@ -59,12 +64,22 @@ export const ComplianceScoreBar = ({
               gap: 1px;
             `}
           >
+            {!totalPassed && !totalFailed && (
```

Comment:
This can only happen when there are unknown findings, for now when there are only unknown findings we represent it as 0% score with gray color bar chart